main.fmf

```diff
@@ -2489,7 +2675,8 @@
         TEST_SUBJECT: TestTuned.testBasic
       non_destructive: true
       skipImage:
+      - rhel-8-3-distropkg
       - fedora-coreos
       summary: check-tuned TestTuned testBasic
-duration: 10m
 test: test/common/run-tests
+duration: 20m
```

Comment:
Duration is certainly not something that belongs into the .fmf file, as it depends on where you run it.

Reply:
you are right, but TMT does that in case duration is not set, so that it set to `5m` it could be probably rewritten in TMT plan, but not sure.
On the other hand, theoretically test itself should be more less the same time duration, what differs is that tests could also do provision part (what could be part of another tool and test just say which resources needs), so that when downloading images or starting VMs it can be much longer or I'm not right with this test time?